torch/csrc/jit/python/python_ivalue.h

```diff
@@ -11,17 +11,21 @@ namespace ivalue {
 struct C10_EXPORT ConcretePyObjectHolder final : PyObjectHolder {
  public:
   static c10::intrusive_ptr<PyObjectHolder> create(py::object py_obj) {
```

Comment:
Since this data wrapper takes care of acquiring GIL on destruction.

I wonder can it also acquiring GIL on creation?

```cpp
static c10::intrusive_ptr<PyObjectHolder> create(const py::object& py_obj) {
py::gil_scoped_acquire ag;
return c10::make_intrusive<ConcretePyObjectHolder>(py_obj);
}
```

Reply:
good point! I will make the change if this looks good to @wanchaol too.